### DISCUSSION TITLE
GRD products from ALOS-2 WideScanSAR data using ISCE

### DISCUSSION THREAD

**Discussion by franoedi** [opera-adt/tophu#35] [Q&A]
Hi, I´m wondering if anyone knows if and how it is possible to generate GRD products from raw ALOS-2 WideScanSAR images using ISCE2?

Regards
Frano

**Comment by scottstanie** [comment]
Hi Frano, I think you may want to ask this in the [isce2 discussion page](https://github.com/isce-framework/isce2/discussions), as this repository is a tool for phase unwrapping
